cpp/oneapi/dal/table/backend/homogen_table_builder_impl.hpp

```diff
@@ -81,12 +82,37 @@ class homogen_table_builder_impl
 
     void copy_data(const void* data, std::int64_t row_count, std::int64_t column_count) override {
         check_copy_data_preconditions(row_count, column_count);
+
+#ifdef ONEDAL_DATA_PARALLEL
+        auto this_q = data_.get_queue();
+        if (this_q) {
+            ONEDAL_ASSERT(is_known_usm(data_));
+            return detail::memcpy_host2usm(*this_q,
```

Comment:
Recommend to use `backend::memcpy_host2usm`

Reply:
discussed offline